GameData/002_CommunityPartsTitles/Localization/CPT_OpenCockpit_loc.cfg

```diff
@@ -29,9 +29,9 @@ Localization
 
 		#LOC_nesdOC_title_OCovhSmall =  Wing Overhead panel Type A (0.5x0.5) (OC)  // Overhead panel Type A (OC)
 		#LOC_nesdOC_title_OCovhBig   =  Wing Overhead panel Type B (1x0.8) (OC)    // Overhead panel Type B (OC)
-		#LOC_nesdOC_title_OCovhBig2  =  Wing Overhead panel Type C (1x0.9) (OC)   // Overhead panel Type C (OC)
+		#LOC_nesdOC_title_OCovhBig2  =  Wing Overhead panel Type C (1x0.9) (OC)
```

Comment:
Why comment is removed?

Reply:
That's an oops. Good catch. I've re-added the comment.